src/com/visualization/piechart/piechart.less

```diff
@@ -21,6 +21,7 @@
 
         & .piechart-svg {
             & .piechart-segment {
```

Comment:
Finally, since the piechart relationship is a given, we can git explicit again. `& .-segment {`

I omitted the `>` here so to not limit how you structure the things inside the SVG. If you are certain it's a direct descendent, you can include it.